src/order/category/HeytAlg.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.BoundedDistribLattice
+import order.heyting.hom
+
+/-!
+# The category of Heyting algebras
+
+This file defines `HeytAlg`, the category of Heyting algebras.
+
+## References
+
+* [nLab, *Frm*](https://ncatlab.org/nlab/show/Frm)
```

Comment:
This reference isn't really appropriate - it doesn't mention Heyting algebras anywhere, so I don't think this is helpful